.gitignore

```diff
@@ -4,7 +4,10 @@ localData/
 dist/
 .yext
 .artifact-output
+.env
+!.env.local
 sites-config/features.json
+functions/functionMetadata.json
```

Comment:
Can you make sure this in all of the starters?

Reply:
Can do!